--- openssl-1.0.2a/ssl/ssl_ciph.c.secure_getenv_mingw	2015-04-24 18:34:32.572168275 +0200
+++ openssl-1.0.2a/ssl/ssl_ciph.c	2015-04-24 18:35:03.060679959 +0200
@@ -484,7 +484,7 @@
             MemCheck_off();
             ssl_comp_methods = sk_SSL_COMP_new(sk_comp_cmp);
             if (ssl_comp_methods != NULL
-                && secure_getenv("OPENSSL_DEFAULT_ZLIB") != NULL) {
+                && getenv("OPENSSL_DEFAULT_ZLIB") != NULL) {
                 comp = (SSL_COMP *)OPENSSL_malloc(sizeof(SSL_COMP));
                 if (comp != NULL) {
                     comp->method = COMP_zlib();
